core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/saved-workflow-section.component.html

```diff
@@ -15,8 +15,8 @@
           <mat-divider></mat-divider>
         </mat-list>
       </div>
-      <button (click)="openNgbdModalAddWorkflowComponent()" class="btn btn-outline-dark add-button" type="button">
-        <i class="fa fa-plus" color="dark"></i></button>
+      <!--      <button (click)="openNgbdModalAddWorkflowComponent()" class="btn btn-outline-dark add-button" type="button">-->
+      <!--        <i class="fa fa-plus" color="dark"></i></button>-->
```

Comment:
why it's commented out?

Reply:
I disabled it intentionally. It does not make sense to have an add button in the dashboard. Or at least the logic should change: for example it should jump directly to the workspace with a new empty workflow loaded. Disabled so it does not confuse the user.

Reply:
if the user is in the dashboard page, and the user wants to create a new workflow, what's the current way to do it?

Reply:
now changed to the following when click add workflow from dashboard:
1. redirect to the workspace with URL: /workflow/new
2. clean cache, undo/redo stack.
3. create a new empty workflow.